src/components/FooterSection.vue

```diff
@@ -71,14 +71,14 @@
         </div>
       </div>
       <div class="column">
-        <aside class="donate-section">
```

Comment:
I guess it would be better to leave the markup of this tag to be aside itself since it follows the rules of good HTML semantics and also helps in improving accessibility.

Reply:
I changed that because OP's csv file reported this issue:https://dequeuniversity.com/rules/axe/3.3/landmark-complementary-is-top-level

I've seen aside used in main all the time, but this aside seems to be contained in a column <div>. Since there's already a container grid structure, I think the aside tag is out of place. It's not a big issue, I will change it back.

I am not that familiar with aria-labels, usually framework handles it, glad you point out my inconsistency, I will change them lower cases and maybe tighten up the naming as well.